fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/cache/BulkDataTenantSpecificCache.java

```diff
@@ -17,13 +16,11 @@
 import com.ibm.fhir.operation.bulkdata.config.BulkDataConfigUtil;
 
 /**
- * This class implements a cache of BulkData configuration organized by tenantId. Each object stored in the cache will be a
+ * This class implements a cache of BulkData configuration organized by tenantId. Each object stored in the cache will
+ * be a
  * two-level map of SearchParameters organized first by resource type, then by search parameter name.
```

Comment:
SearchParameters or jobParameters?

Reply:
funny - we never caught this in the initial implementation. I'll fix. It's actually suppose to say a map of properties related to the bulkdata export.